scripts/generateManifestForRustAdblock.js

```diff
@@ -38,9 +39,10 @@ const generateManifestFileForDefaultAdblock =
 
 const generateManifestFilesForAllRegions = () => {
   let p = Promise.resolve()
-  new lists('regions').forEach((region) => { // eslint-disable-line
-    p = p.then(generateManifestFile.bind(null, region.title,
-      region.base64_public_key, region.uuid))
+  getRegionalLists().then(regions => {
+    regions.forEach((region) => {
+      p = p.then(generateManifestFile.bind(null, region.title, region.base64_public_key, region.uuid))
```

Comment:
do you mean to be creating a new instance of the `generateManifestFile` function here on each iteration of the forEach loop?  Would be easier / cheaper to just call the single version of `generateManifestFile` with different params I think

Reply:
That's also copied that from what it used to be, not sure how necessary it was